index.js

```diff
@@ -26,18 +27,37 @@ const stripCookieHeader = header => {
 	return header.substring(start, end + 1);
 };
 
+const parseCookieName = cookieName => {
+	const nameMatch = COOKIE_NAME.exec(cookieName);
```

Comment:
The name is the entire thing, so the group can be removed from the `COOKIE_NAME` regex and this function can be simplified to

```js
cookieName.test(cookieName) ?
	cookieName :
	null
```

… which is a bit of an unusual implementation. Maybe it should export `isCookieName` instead, even though it’s less uniform?

Reply:
Good point, cleaned that up.